cmd/crc/cmd/podman_env.go

```diff
@@ -33,18 +37,24 @@ func runPodmanEnv() error {
 		return err
 	}
 
+	socket := "/run/podman/podman.sock"
+	if rootless {
+		socket = "/run/user/1000/podman/podman.sock"
+	}
```

Comment:
This is changing the socket which is used by default, not clear to me why we'd want to use one rather than the other?